app/views/devise/registrations/new.html.erb

```diff
@@ -7,7 +7,7 @@
     <%= f.input :email, required: true, autofocus: true %>
     <%= f.input :password, required: true, hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length) %>
     <%= f.input :password_confirmation, required: true %>
-    <%= f.input :bot_field %>
+    <%= f.input :bot_field, label: "Fill out this field if you are a bot" %>
```

Comment:
Make it clear to users (screen readers?) that this field shouldn't be filled out normally.